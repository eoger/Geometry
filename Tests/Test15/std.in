C cercle 0 0 5
R rect88 12 12 24 24 
L l42 42 42 42 42
OA 1oa cercle rect88
PL plpl 10 10 9 9 52 52 1 1
LIST
MOVE cercle 1 -1
MOVE rect88 -12 -12
MOVE l42 -84 -84
MOVE plpl -10 -0
LIST
MOVE 1oa -1 1
LIST
OA 3oa rect88 l42 plpl
MOVE 3oa 0 30
LIST
OA 666oa cercle 3oa 1oa
MOVE 666oa 700 0
LIST
MOVE cercle 0 0
LIST
EXIT