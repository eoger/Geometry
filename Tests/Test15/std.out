OK
OK
OK
OK
OK
OA 1oa cercle rect88
C cercle 0 0 5
L l42 42 42 42 42
PL plpl 10 10 9 9 52 52 1 1
R rect88 12 12 24 24 
OK
OK
OK
OK
OA 1oa cercle rect88
C cercle 1 -1 5
L l42 -42 -42 -42 -42
PL plpl 0 10 -1 9 42 52 -9 1
R rect88 0 0 12 12
OK
OA 1oa cercle rect88
C cercle 0 0 5
L l42 -42 -42 -42 -42
PL plpl 0 10 -1 9 42 52 -9 1
R rect88 -1 1 11 13
OK
OK
OA 1oa cercle rect88
OA 3oa l42 plpl rect88
C cercle 0 0 5
L l42 -42 -12 -42 -12
PL plpl 0 40 -1 39 42 82 -9 31
R rect88 -1 31 11 43
OK
OK
OA 1oa cercle rect88
OA 3oa l42 plpl rect88
OA 666oa 1oa 3oa cercle
C cercle 700 0 5
L l42 658 -12 658 -12
PL plpl 700 40 699 39 742 82 691 31
R rect88 699 31 711 43
OK
OA 1oa cercle rect88
OA 3oa l42 plpl rect88
OA 666oa 1oa 3oa cercle
C cercle 700 0 5
L l42 658 -12 658 -12
PL plpl 700 40 699 39 742 82 691 31
R rect88 699 31 711 43
